### PR TITLE
BZ 1194479 test

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 189256e1747d61ee6b0190b37209ceb14e9481b1:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c74e7e78756f801f0eb128ab04f3ed650d4a93b6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/services/test_request.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            continue
+
+        new_idlist.append(idlist[i])
+        new_argvalues.append(argvalues[i])
+
+    testgen.parametrize(metafunc, argnames, new_argvalues, ids=new_idlist, scope="module")
+
+
+@pytest.fixture()
+def provider_init(provider_key):
```
Comment (seandst) [inline cfme/tests/services/test_request.py]: Almost missed this...

`provider_init` should go away, use [setup_provider](https://github.com/RedHatQE/cfme_tests/blob/master/fixtures/provider.py#L43) instead.

**Comment by dajoRH** [comment]
Lint report for commit 08ee0574be70635b76ad51591fd9f44159f15d1d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7adeba42827a400fffd9aa186f50487f0a42f67f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
Is it supposed to skip everywhere? I'd personally prefer the `.format()` syntax over `%` syntax in string formatting. Apart from that it looks good.

**Comment by dajoRH** [comment]
Lint report for commit 3ca9fef5cfd5271eab8a09cd81c4bb07401ed583:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d225bb87721c4f2286811f8e804054f9e7503982:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 16c90c25b5c4aa2f77d121457afbf5d9fff7a6b6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 670a06b7de3208ccb2012b5cc1d7b582f304344c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 660d077032bd85c0c637313db0c5ab59e90d621a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
@sshveta I see you kee pushing new commits, so ping me then when you are happy with it.

**Comment by sshveta** [comment]
Please merge this PR.

**Comment by mfalesni** [comment]
:+1: :cookie: :facepunch:

**Pull request merged**